logic/transactionPool.js

```diff
@@ -101,7 +101,9 @@ TransactionPool.prototype.transactionInPool = function (id) {
 		self.bundled.index[id],
 		self.queued.index[id],
 		self.multisignature.index[id]
-	].filter(Boolean).length > 0;
+	].filter(function (index) {
+		return typeof(index) === 'number';
```

Comment:
I would suggest to use `some` as it returns a boolean and does not iterate through a whole array if finds a match before:
`[...].some(function(index){return typeof index === 'number'})`